prov/verbs/src/fi_verbs.c

```diff
@@ -1744,7 +1744,16 @@ fi_ibv_msg_ep_getopt(fid_t fid, int level, int optname,
 {
 	switch (level) {
 	case FI_OPT_ENDPOINT:
-		return -FI_ENOPROTOOPT;
+		switch (optname) {
+		case FI_OPT_CM_DATA_SIZE:
+			if (*optlen < sizeof(size_t))
+				return -FI_ETOOSMALL;
+			*((size_t *) optval) = 56;
```

Comment:
I had to read the IB spec, calculate the size of the RDMA CM header, and this is the size that was left.  :P